dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/DiscoverResultConverter.java

```diff
@@ -45,6 +46,9 @@
     @Autowired
     private List<DSpaceObjectConverter> converters;
 
+    @Autowired
+    private ItemService itemService;
+
```

Comment:
This is currently not required here, or am I missing something?

Reply:
That’s correct, it will be removed

Reply:
This should be resolved using commit b37d0e216718316ba202f34556f53344cd0346b8